tests/host_task/cuda_host_task_interop_api.cpp

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification for sycl::interop_handle::get_native functions
+//  this test check interop API with CUDA back-end only.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+#ifdef SYCL_BACKEND_CUDA
+
+#include <cuda.h>
+#include <sycl/ext/oneapi/experimental/backend/cuda.hpp>
+
+#endif  // SYCL_BACKEND_CUDA
+
+#define TEST_NAME cuda_host_task_interop_api
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+
+class TEST_NAME : public sycl_cts::util::test_base {
```

Comment:
Please, update this test to use Catch2 framework API directly.

Reply:
I have updated the cuda host_task test to use Catch2